packages/navie/src/services/vector-terms-service.ts

```diff
@@ -90,15 +85,20 @@ Terms: test cases +logContext jest`,
     role: 'user',
   },
   {
-    content: `Context: auth authentication authorization
-Instructions: Describe the authentication and authorization process
----
-Terms: +auth authentication authorization token strategy provider`,
+    content: JSON.stringify({
+      context: 'auth authentication authorization',
+      instructions: 'Describe the authentication and authorization process',
+      terms: ['+auth', 'authentication', 'authorization', 'token', 'strategy', 'provider'],
+    }),
     role: 'assistant',
   },
 ];
 
-const parseText = (text: string): string[] => text.split(/\s+/);
+const schema = z.object({
+  context: z.string(),
+  instructions: z.string(),
+  terms: z.array(z.string()),
```

Comment:
The description of these properties could move out of the prompt and in to the schema

Reply:
True, although I tried to change it as little as possible as not to break anything.